### PR TITLE
WIP Build Cleanup

### PR DESCRIPTION
**Pull request by krisselden**

### CODE REVIEW

File: ember-cli-build.js [hunk 1]
```diff
@@ -1,305 +1,504 @@
...
-      cache: null,
-      sourceRoot: '/'
-    }
+function buildLib(name, entry, libPath) {
+  var dir = path.resolve(path.dirname(require.resolve(name)), libPath);
+  return new Rollup(dir, {
+    rollup: {
+      entry: entry,
+      plugins: [{
+        load: function (id) {
```
Comment (stefanpenner) [inline ember-cli-build.js]: can we extract this plugin into a separate file?

File: ember-cli-build.js [hunk 2]
```diff
@@ -1,305 +1,504 @@
...
+      } else if (include.indexOf(importee) > -1) {
+        return root + importee + '/index.js';
+      }
+    }
+  };
+}
 
-  return merge(finalTrees);
-};
+//   var benchmarkPath = __dirname + '/node_modules/benchmark';
```
Comment (stefanpenner) [inline ember-cli-build.js]: likely needs cleanup?

**Comment by krisselden** [comment]
@stefanpenner the idea would be to just rollup the es6 dist from here into ember-glimmer.

**Comment by tomdale** [comment]
Closing in favor of the new stuff that uses `broccoli-typescript-compiler@2`.

**Pull request closed without merging**